pkg/image/apis/image/v1/helpers.go

```diff
@@ -113,10 +113,12 @@ func ImageWithMetadata(image *Image) error {
 	case 1:
 		image.DockerImageManifestMediaType = schema1.MediaTypeManifest
 
-		if len(manifest.History) == 0 {
```

Comment:
@legionus the manifest check at the top of ImageWithMetadata will return `nil` in that case so we don't have the metadata... I wonder why the manifest is empty in this case.